### DISCUSSION TITLE
如果你更换到 Dropbox 会引起 API 无法使用的问题

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#2281] [Developer]
> ## 1. 如果发生 **Dropbox 无法授权的解决方案，包括：闪退与提示密码过长**，请看 [解决方案 ](https://github.com/Kenshin/simpread/discussions/3171)

> ## 2. 注意：2021年10月5日开始，Dropbox 功能已经恢复，但因为复杂的网络环境，建议仍选用坚果云方案。

> ## 3. 如仍发生下面的问题，请更换坚果云方案。

***

## 问题现象

> 发生问题的 **仅为 Dropbox 作为同步盘** 的情况，如果 **使用坚果云，则没有这个问题**。

> 如使用 Telegram bot 保存 URL 的话，如果提示，基本上就可以判定此原因。

![image](https://user-images.githubusercontent.com/81074/120056652-4ddaff00-c070-11eb-9c62-1f1e2f488422.png)

## 解决方案

> 将 **同步盘切换为坚果云**，即可解决。

## 原因分析

> 因 **访问 Dropbox 需要保证网络畅通**，所以简悦使用了一个 **中转服务器**，近期由于某些原因导致中转服务器也需要保证网络畅通，因此导致此问题。

## 何时解决

> 因并不是简悦本身的问题，所以等 中转服务器 网络畅通后，自行解决此问题。  😂

**Comment by ImSingee** [comment]
要不然套一层 Cloudflare … Vercel 确实不太稳

**Reply by Kenshin** [reply]
emmm 其实可以这么干。
等我抽出时间弄下~
